relay-server/src/metrics_extraction/spans/mod.rs

```diff
@@ -455,3 +455,271 @@ fn normalize_domain(domain: &str, port: Option<&String>) -> Option<String> {
     }
     Some(replaced)
 }
+
+#[cfg(test)]
+mod tests {
+    use relay_general::types::Annotated;
+
+    use relay_general::store::{self, LightNormalizationConfig};
+    use relay_metrics::AggregatorConfig;
+
+    use crate::metrics_extraction::spans::extract_span_metrics;
+
+    #[test]
+    fn test_extract_span_metrics() {
+        let json = r#"
```

Comment:
I thought about extracting this json into a file as the test is already a bit hard to read as-is. Thoughts?